### PR TITLE
Update bitlocker-management.md

### PR DESCRIPTION
**Pull request by NCBrady**
added some clarity on Windows information protection versus WIP

### CODE REVIEW

**Comment by PRMerger14** [comment]
@NCBrady : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/protect/plan-design/bitlocker-management.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Starting in version 1910, use Configuration Manager to manage BitLocker Drive En
 > [!Note]  
 > Configuration Manager doesn't enable this optional feature by default. You must enable this feature before using it. For more information, see [Enable optional features from updates](/configmgr/core/servers/manage/install-in-console-updates#bkmk_options).  
 
-For more information on Windows information protection, see [BitLocker overview](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview).
+For more information on Windows "information protection" (not to be confused with Windows Information Protection or WIP (https://docs.microsoft.com/en-us/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip), see [BitLocker overview](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview).
```
Comment (aczechowski) [inline sccm/protect/plan-design/bitlocker-management.md]: How about just simplifying to something like the following? 😉 

```suggestion
For more information, see [BitLocker overview](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview).
```

**Comment by PRMerger9** [comment]
@NCBrady : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @NCBrady! 😄 
#sign-off
*Once merged*, this should be live with the next daily publishing process. Right now that *could* be today around 11:00 Pacific time. (If not today, then tomorrow same time.) No further action needed from you.

**Pull request merged**